drivers/ipvlan/ipvlan_endpoint.go

```diff
@@ -28,9 +28,9 @@ func (d *driver) CreateEndpoint(nid, eid string, ifInfo driverapi.InterfaceInfo,
 	}
 	ep := &endpoint{
 		id:     eid,
+		nid:    nid,
 		addr:   ifInfo.Address(),
 		addrv6: ifInfo.AddressIPv6(),
-		mac:    ifInfo.MacAddress(),
```

Comment:
Because the code before of this. If libnetwork passes a non nil mac to the driver, the driver fails to start.